test/db/archos/darwin-x64/cmd_i

```diff
@@ -7,7 +7,7 @@ e io.cache=true
 "(show_fname fname; wz $0; wtf `prz`; o `prz`; i~file; ij; rm `prz`)"
 .(show_fname A\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
 .(show_fname B\x1b¢\302\200€𝄞\363\240\201\201\\.bin)
-# INVALID FILENAME .(show_fname B\x1b¢\302\200\200€𝄞\363\240\201\201\\.bin)
+# INVALID FILENAME .(show_fname B\x1b¢\302\200\201€𝄞\363\240\201\201\\.bin)
```

Comment:
I wonder why this change?

Reply:
This is just so that `archos/linux-x64/cmd_i` and `archos/darwin-x64/cmd_i` are in sync